### ISSUE TITLE
[Feature] Startpage for image search

### ISSUE THREAD

**Issue by Apacelus** [searxng/searxng#2540]
**Working URL to the engine**
https://startpage.com

**Why do you want to add this engine?**
Using Google images directly leads to tracking and personalized search results (even if not logged in), which is especially bad for instances with only a few users. Yet the results from Google are pretty good.
Solution: Use Startpage as a proxy to avoid tracking. 

**Features of this engine**
Google images without tracking

**Applicable category of this engine**
images

**Additional context**
Startpage is already available as a general search engine, but not as an image search engine.

**Comment by xbdmHQ** [comment]
> **Working URL to the engine** https://startpage.com
> 
> **Why do you want to add this engine?** Using Google images directly leads to tracking and personalized search results (even if not logged in), which is especially bad for instances with only a few users. Yet the results from Google are pretty good. Solution: Use Startpage as a proxy to avoid tracking.
> 
> **Features of this engine** Google images without tracking
> 
> **Applicable category of this engine** images
> 
> **Additional context** Startpage is already available as a general search engine, but not as an image search engine.

I think you may be confused, there is already an image proxy in searxng already.

**Comment by Apacelus** [comment]
1. User enters search query
2. Searxng instance searches for images with Google; tracking and personalization happens
3. Searxng delivers the personalized (by Google, not the instance) results to the user
4. User clicks on image
Is step 4 where the proxy is? And what does it proxy through?

**Comment by return42** [comment]
Image proxy is active in most instances (docker image and default script installations) // can be enabled/disabled by [`image_proxy`](https://docs.searxng.org/admin/settings/settings_server.html) .. I think a instance that does not activates the image proxy is not really privacy protecting.

> Is step 4 where the proxy is? And what does it proxy through?

As long as the user navigates in the SearXNG result page, the images are coming from the SearXNG instance which acts as an proxy for the images .. when the user clicks on a link or image in the result list, the origin URL is loaded.

Even if your argumentation for Startpage-Images is already covered by the image proxy, I think it makes sense to implement the other Startpage services like images, videos and news as engines in SearXNG.

I had also thought about it for a while, but unfortunately not yet had time to implement it / another problem that may need to be solved first are the constant CAPTCHAS of Startpage.

We should leave this issue open, the requirement is valid, even if the argumentation of the opening comment is not quite true.

**Comment by Apacelus** [comment]
So the image proxy is just the instance? That makes it basically useless for instances that have a very small amount of users, as I mentioned in my initial comment:
> which is especially bad for instances with only a few users

That is why I made the feature request, as I am the sole user of my instance and therefore searching for images directly via Google (through the searxng instance) leads to tracking, as the instance always uses the same ip address.

**Comment by return42** [comment]
> via Google (through the searxng instance) leads to tracking, as the instance always uses the same ip address.

This is the same with every query term! .. but even if the IP is the same on every request, at least no cookies are stored by Google & Co and no CSP is used and no JS tracking tools .. and all that bloat ... But to get real privacy mix your queries with those from other users: make your engine public or use a public one that already exists --> https://searx.space/

**Comment by Apacelus** [comment]
> This is the same with every query term!

There is no tracking if my instances uses private/proxy search engines only, i.e. brave search, startpage, duckduckgo.

> at least no cookies are stored by Google & Co and no CSP is used and no JS tracking tools

That sure does make it harder for those services to track an instance

> use a public one that already exists 

The reason why I run my own, "private" instance, is that the public instances seem to get tracked after a while. I somehow start getting the one same search result, no matter the query, as well as lots of other "weirdness" in the search results (like search results that are barely close to the query).

Also, the instances arent very reliable and all get that weirdness after some time.

**Comment by return42** [comment]
> The reason why I run my own, "private" instance, is that the public instances seem to get tracked after a while. 

I don't think that instances with a broad user base can become a information bubble / https://searx.be/ is an instance with a broad user base.

> I somehow start getting the one same search result, no matter the query, as well as lots of other "weirdness" in the search results (like search results that are barely close to the query).
> Also, the instances arent very reliable and all get that weirdness after some time.

Sounds strange .. never seen / not sure what you mean .. can you please test my instance for a while and give me feedback if you have same issues on my instance --> https://darmarit.org/searx/

**Comment by unixfox** [comment]
> There is no tracking if my instances uses private/proxy search engines only, i.e. brave search, startpage, duckduckgo.

These websites can also track you, you have to entirely trust them to not track you if you are a single user using a private instance.

One way to somewhat solve this issue is to pass the searxng traffic through a shared IP like a VPN or a mobile connection data or a shared proxy.
